conans/client/graph/compute_pid.py

```diff
@@ -67,6 +67,7 @@ def compute_package_id(node, new_config, config_version):
                     # This 'cant_build' will be ignored if we don't have to build the node.
                     node.cant_build = str(e)
 
+    conanfile.settings._frozen_rm = True
```

Comment:
I suppose that as this is at the end of most processes, there is no need to set this back to true right?

Reply:
No, after this point there is no reason to remove a setting in the recipe. Actually it could be done earlier, after ``configure()``, but just in case.